eox_hooks/edxapp_wrapper/backends/enrollments_l_v1_test.py

```diff
@@ -0,0 +1,13 @@
+"""
+Backend file for Course Enrollment related objects.
+"""
+
+
+def get_course_enrollment():
+    """
+    Gets Course Enrollment class model from edxapp.
+
+    Returns:
+        [Class]: Course Enrollment edxapp model.
+    """
+    return object
```

Comment:
About the backends:
I don't think you need to change the backends we don't use (ex: course_mode_j, courses_j, models_j, etc). But you already did it. It's okay.
About the test backends, I suggest using something like [Mock](https://docs.python.org/3/library/unittest.mock.html#the-mock-class) instead of object:
```suggestion
    return Mock()
```
Mock is an object you can use for tests. The mock objects allow you to call it, know if you are doing the right calls, and are flexible enough to modify it if you need it. 

Let me know your thoughts.